cli/src/utils/loadLocalCommands.ts

```diff
@@ -4,8 +4,34 @@ import Ora from "ora";
 import * as glob from "glob";
 import { readPackageJSON } from "./readPackageJSON";
 import { ensureNoTsHeaderFiles } from "./ensureNoTsHeaderFiles";
+import { Command } from "commander";
 
-export async function loadLocalCommands(program): Promise<boolean> {
+// We don't want to require actionhero in the project (so we can rely on core's version), so we need to stub this type
```

Comment:
While I don't think it's worth the dev time right now.
[deno](https://deno.land/) is above 1.0 and I think [could import this type from a url](https://deno.land/manual@v1.14.3/linking_to_external_code)
Since our CLI is somewhat limited in scope it's potentially a good place to try this out.

Reply:
Heh - I don't think changing how our customers install and deploy things is in scope for this PR :D